src/components/BaseStyles/BaseStylesService.spec.js

```diff
@@ -13,12 +13,13 @@
  * limitations under the License.
  */
 
-import { circuit } from '../../themes';
+import { light } from '@sumup/design-tokens';
```

Comment:
Did you also update the deprecated `GlobalStyles` component? Or will we remove it in v2 when introducing external tokens?

Reply:
The `GlobalStyles` component was renamed to `BaseStyles` to make its purpose clearer. The base styles are mostly style resets for a consistent cross-browser appearance. They use the new design tokens but are otherwise unrelated and specific to the web platform.